phpunit

```diff
@@ -4,7 +4,8 @@ set -o errexit
 set -o nounset
 
 check_phpunit_version() {
-    INSTALLED_PHPUNIT_VERSION=$(composer info phpunit/phpunit | grep versions |  cut -c 14-)
+    COMPOSER_BIN=$(which composer)
+    INSTALLED_PHPUNIT_VERSION=$(php -d xdebug.mode=off ${COMPOSER_BIN} info phpunit/phpunit | grep versions |  cut -c 14-)
```

Comment:
this has nothing to deal with this fixture, but when using step-by-step debugger, xdebug was stopping this command